Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -137,5 +138,12 @@ private void ResetPlayerToOrigin()
         characterController.enabled = false;
         transform.position = playerResetPosition;
         characterController.enabled = true;
+    }
+
+    private void HaltPlayer()

```

Comment:
This is here because when isPaused is set, the movement vector will remain as it was pre-canvas open (set in Update). So when they exit the canvas, this might cause some unintended behaviour? Pretty minor lol so can get rid of this if you want.

Reply:
it doesn't look like this does anything, i just tested it. you're good to delete it